stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -72,19 +72,25 @@ class Pool():
     def __exit__(self, exc_type, exc_val, exc_tb) -> None: ...
 
 class Process():
+    name: str = ...
+    daemon: bool = ...
+    pid: Optional[int] = ...
+    exitcode: Optional[int] = ...
+    authkey: bytes = ...
+    sentinel: int = ...
     # TODO: set type of group to None
     def __init__(self,
-                 group: Any = ...,
+                 group: Any = None,
```

Comment:
We actually prefer using `...` for all default values. In stubs, it doesn't matter exactly what the default is.